demo/config.yaml

```diff
@@ -108,6 +108,6 @@ lcr-modules:
         inputs:
             sample_bam: "data/{sample_id}.bam"
             sample_bai: "data/{sample_id}.bam.bai"
-        scratch_subdirectories: ["mpileup"]
-        unmatched_normal_ids:
-            genome--hg38: "BLGSP-71-23-00361-14A-01E"
+        scratch_subdirectories: [] # mpileup should be in scratch space
+        unmatched_normal_ids: 
+            genome--hg38: "BLGSP-71-23-00361-14A-01E" ## FFPE normal
```

Comment:
Can you delete these two lines please? Usually the unmatched normal ID is set in the shared lcr-modules config, and users can set module-specific unmatched_normal_ids in their project config, too. We don't want this in the module since users at other sites won't have this normal.

Reply:
Will do!